src/mapml/layers/TemplatedFeaturesLayer.js

```diff
@@ -70,11 +70,18 @@ export var TemplatedFeaturesLayer = Layer.extend({
         projection: map.options.projection,
         mapEl: this._linkEl.getMapEl(),
         onEachFeature: function (properties, geometry) {
+          const popupOptions = {
```

Comment:
I believe the autoPan value was the default `false` and that was intentional so that the map wouldn't keep panning and pulling in new features, which might be a problem for a screenreader user (?).  That was my theory anyhow.  I guess we can keep this value and see if it engenders any comment from anyone else though.